client/create.go

```diff
@@ -24,16 +34,25 @@ For more about knox, see https://github.com/pinterest/knox.
 See also: knox add, knox get
 	`,
 }
+var createTinkKeyset = cmdCreate.Flag.Bool("key-template", false, "")
```

Comment:
this should be a string argument instead of a boolean, right?

Reply:
Thank you so much! I have revised my code to use **Flag.String** here.